content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -174,6 +174,49 @@ spec:
 
 However, the particular path specified in the custom recycler Pod template in the `volumes` part is replaced with the particular path of the volume that is being recycled.
 
+### Pre-binding
+
+The controller in the master can [bind PVCs to matching PVs](#binding) in the
+cluster. However, if you want a PVC to bind to a specific PV, you need to
+pre-bind them. As it's a bi-directional binding between the PVC and the PV,
+there are two ways:
+
+- Specify the PV in your PVC using the `volumeName` field
+- Specify the PVC in your PV using the `claimRef` field
+
+Normally, you can only specify the PV in your PVC, then the PVC will only be
+able to bind to this specific PV. If the PV exists and is `Available`, the PV
+and PVC will be bound regardless of whether the PV satisfies the matching
+criteria, e.g. node selector, storage request.
+
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: foo-pvc
+  namespace: foo
+spec:
+  volumeName: foo-pv
+  ...
+```
+
+If the PV may be used by other PVCs, you need to reserve it by specifying the
+PVC in the PV using the `claimRef` field so that other PVCs can not bind to it.
+
+```
+apiVersion: v1
+kind: PersistentVolum
+metadata:
+  name: foo-pv
+spec:
+  claimRef:
+    name: foo-pvc
+    namespace: foo
+  ...
+```
+
+This is useful if you want to consume PVs of which the data is retained before.
```

Comment:
```suggestion
This is useful if you want to consume PVs that already exist.
```

Reply:
there are two scenarios:

1) PVs are statically-provisioned and are empty
2) PVs are statically-provisioned or dynamically-provisioned but are retained (`claimPolicy` is `Retain`), so that they can be reused by other PVCs

I want to emphasize the latter. As for the former, users don't need to pre-bind the PVC and the PV normally.

Reply:
How about:
```suggestion
This is useful if you want to consume PersistentVolumes that have their `claimPolicy` set
to `Retain`, including cases where you are reusing an existing PV.
```
?